mergify_engine/worker.py

```diff
@@ -489,6 +494,16 @@ async def _consume_buckets(
             else:
                 break
 
+            if (time.monotonic() - started_at) >= config.BUCKET_PROCESSING_MAX_SECONDS:
+                low_prio_threshold = get_low_priority_minimal_score()
+                if _bucket_score < low_prio_threshold:
+                    statsd.increment("engine.buckets.preempted.high-prio")
+                else:
+                    statsd.increment("engine.buckets.preempted.low-prio")
+                break
```

Comment:
```suggestion
            if (time.monotonic() - started_at) >= config.BUCKET_PROCESSING_MAX_SECONDS:
                low_prio_threshold = get_low_priority_minimal_score()
                prio = "high" if _bucket_score < low_prio_threshold else "low"
                statsd.increment("engine.buckets.preempted", tags=[f"priority:{prio}"])
                break
```

I feel like we should also change the other metrics to set the priority as a tag.